src/extension.ts

```diff
@@ -122,7 +128,8 @@ function openTemplateAndSaveNewFile(type: string, namespace: string, filename: s
         .then(doc => {
             let text = doc.getText()
                 .replace('${namespace}', namespace)
-                .replace('${classname}', filename);
+                .replace('${classname}', filename)
+                .replace('${enumname}', filename);
```

Comment:
Why use 'enumname' here, when in both the other template files, 'classname' is used?
I like consistency. Why not use 'classname', or change the interface one to 'interfacename', or change all to something like 'itemname' or just 'name'?

Reply:
You're right. I've more or less auto-merged the commits by the original author, Venipa, and I missed that when fixing the merge conflicts. I've pushed a more consistent version. Thanks.